client/src/components/Shipping/ShippingOptions.jsx

```diff
@@ -0,0 +1,21 @@
+import moment from 'moment';
+import React from 'react';
+
+function ShippingOptions({ details }) {
+  if (details && details[0].fastShipping) {
+    var newDate = moment().add(2, 'day').format('dddd, MMMM Do');
```

Comment:
Great job with this, I was struggling on how to create a new date 2 days from today, ended up having to do more work than using the moment library!